packages/cisco_asa/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -1192,6 +1205,19 @@ processors:
         CISCO_USER: (?:\*\*\*\*\*|(?:(?:LOCAL\\)?(?:%{HOSTNAME}\\)?%{USERNAME}\$?(?:@%{HOSTNAME})?%{CISCO_SGT}?)|[^$]+)
         CISCO_SGT: (?:, *)?%{NUMBER}(?::%{WORD})?
 
+  #
+  # In some cases, the ctx._temp_.cisco.message_id field is created as an empty string. This generally
+  # indicates a security event in an older FTD version.
+  #
+  # However, if misconfigured and incorrect data is sent through this pipeline, it is important to
+  # throw an error that is explanatory to the failure condition.
+  - fail:
+      if: '["430001", "430002", "430003", "430004", "430005", ""].contains(ctx._temp_.cisco.message_id) && !ctx.message.contains(":")'
+      message:  >
+          The message_id ( "{{{ctx._temp_.cisco.message_id}}}" ) indicates this to be a security event, but the message field is not formatted appropriately.
+          I am expecting a comma-separated sequence of 'key: value' pairs.
```

nit: Prefer not to say "I" in message.
```suggestion
         Expected a comma-separated sequence of 'key: value' pairs.
```

---

I think I might also want to just remove the actual message ids, since the error happens when it's a `""`